src/main/java/de/tum/in/www1/artemis/repository/ComplaintResponseRepository.java

```diff
@@ -22,4 +22,9 @@
      * @return number of complaints response associated to course courseId
      */
     long countByComplaint_Result_Participation_Exercise_Course_Id(Long courseId);
+
+    /**
+     * Delete all complaint responses that belong to complaints of submission results of a given participation
+     */
+    void deleteByComplaint_Result_Participation_Id(Long exerciseId);
```

Comment:
Why the param is called `exerciseId`? Shouldn't be `participationId`?

Reply:
You are right, in my first approach I used the exerciseId here. I updated it.